src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -22,7 +22,7 @@ def load_command_table(self, _):
     )
     with self.command_group('connectedk8s', connectedk8s_sdk, client_factory=cf_connected_cluster) as g:
         g.custom_command('connect', 'create_connectedk8s', supports_no_wait=True)
-        g.custom_command('update', 'update_agents')
+        g.custom_command('update', 'update_agents_or_resource', supports_no_wait=True)
```

Comment:
do we need to explicitly call out agent vs resource? Can't we use generic term like update CC properties?

Reply:
ack, will make it generic. However, the function name is internal and not visible to customers

Reply:
If I recall, we are adding similar msg in cli help msg.

Reply:
fixed the function name and help message as well